Source/WebCore/style/values/primitives/StylePosition.cpp

```diff
@@ -38,6 +38,83 @@ namespace Style {
 
 using namespace CSS::Literals;
 
+enum XAxisPhysicalKeywords : bool { Left, Right };
+enum YAxisPhysicalKeywords : bool { Top, Bottom };
```

Comment:
Do we need to introduce these enums, or can we re-use `BoxSides`?

Reply:
These avoid the need to have invalid branches, since these functions can only return a subset.

Reply:
That said, I removed the functions and their uses entirely :).